_posts/2020-04-29-release-R119/index.md

```diff
@@ -0,0 +1,120 @@
+
+We, our open source community and customers extensively tested all assets from R118 and now we're excited to announce that R119 marks general availability of the new format and all associated functionality.
+
+We identified several bugs in R118 that are related to changes we've made there:
+
+* Events from a POST payload could get lost if the payload contained at least one corrupted event ([#4320][issue-4320])
+* Enrich process could crash with `NullPointerException` in case of empty query parameter in IgluAdapter ([#4330][issue-4320])
+* Enrich process could crash with `NullPointerException` in case of empty query parameter in Snowplow Adapter ([#4324][issue-4324], thanks [Rob Kingston][robkingston] for spotting it!)
+
+In order to make R119 production-ready we also wanted to make sure that new functionality related to bad rows is on feature-parity (or above) with the legacy format.
+
+Hence the Event Recovery job 0.2.0 will be announced soon, which leverages all benefits of the new format.
+
+<h2 id="monorepo">2. Change in Snowplow Open Source</h2>
+
+Historically, most of Snowplow's Open Source estate was hosted by a single [GitHub monorepository][monorepo].
+This repository holds the whole history of changes back to inception in 2012.
+Almost every month we then would do a big bang release with all assets and an associated blog post.
+One of the goals of these big bang releases was a compatibility guarantee, which meant that all assets within a single release are compatible with each other.
+
+However, as Snowplow's OSS estate was growing, we started to realise that it's getting harder to maintain this guarantee and that the monorepository approach makes our development process very inflexible as we sometimes had to do an umbrella release and a blog post just for an urgent hot fix.
+In order to solve that problem we've made a decision to split the Snowplow monorepository into individual repositories, each containing code for a single application.
+
+We expect this change will make development even more OSS-friendly and easier to contribute to.
+
+We also would like to use this as a next step in our batch pipeline deprecation process and archive the following components:
+
+* [Snowplow Cloudfront Collector][clojure]
+* [Snowplow Clojure Collector][clojure]
+* [Hadoop Elasticsearch Sink][hadoop]
+
+We're creating the following new repositories for our subprojects:
+
+* Scala Stream Collector
+* Scala Common Enrich
+* Stream Enrich
+* Beam Enrich (which already existed as independepent asset for a couple of months)
+* EmrEtlRunner
+* Spark Enrich
+
+We're not going to delete the current monorepository, rather we are planning to use it as starting point for the OSS community, containing all necessary links to the latest assets in a form of git sumbodules.
+
+We will keep doing the Snowplow umbrella meta releases, announcing any breaking changes in the way components work together.
```

Comment:
Not sure why it's not letting me make suggestions... I'd replace this with something like:

> We will be establishing an alternative, better way to communicate which versions are compatible and our recommendation for the stack you should be running - while increasing our ability to get changes to you quickly.

I'm not sure they'll look like umbrella releases as we've had them now. I see them being more rolling, no announcement, etc.